### PR TITLE
Add user-guide and update README.

### PR DESCRIPTION
**Pull request by cdrage**
Updates the readme to use `kedge deploy` as the default command. Also
adds an elementary user guide.

This is a push towards having the website up-and-running for Kedge.

### CODE REVIEW

**Comment by cdrage** [comment]
ping @kadel @surajssd @pradeepto

File: docs/user-guide.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# User Guide
+
+- CLI
+  - [`convert`](#kedge-convert)
```
Comment (kadel) [inline docs/user-guide.md]: there is no `convert` command

File: docs/user-guide.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  - [`completion`](#kedge-completion)
```
Comment (kadel) [inline docs/user-guide.md]: there is no `completion` command

File: docs/user-guide.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  - [`validate`](#kedge-validate)
```
Comment (kadel) [inline docs/user-guide.md]: there is no `validate` command

File: docs/user-guide.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
+# User Guide
+
+- CLI
```
Comment (kadel) [inline docs/user-guide.md]: missing `deploy` and `generate`

File: docs/user-guide.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+Generation commands can also be "piped" to Kubernetes
+
+```sh
+kedge generate -f httpd.yaml | kubectl create -f -
+deployment "httpd" created
+service "httpd" created
+```
+
+## `kedge deploy`
```
Comment (kadel) [inline docs/user-guide.md]: It would be better to put `deploy` in the first place.

**Comment by cdrage** [comment]
@kadel Woops. I forgot to update the "index" part at the top (was copied from Kompose user guide).

I've gone ahead and updated the "index" as well as moved the deploy section as the first part and generate as the second

**Pull request merged**